user-guide/Advanced_Modules/DOM/DOM_DevOps_tools.md

```diff
@@ -0,0 +1,27 @@
+---
+uid: DOM_DevOps_tools
+---
+
+# DOM DevOps tools
+
+The DOM DevOps tools allow you to create and update DOM models in a more graphical way instead of doing so via code. They allow you to quickly model DOM objects even if you do not have a profound knowledge of the different objects and how they are structured in the background.
+
+> [!CAUTION]
+> These tools have been created by developers for developers. We have made them available in case this could be useful, but they are currently by no means complete tools.
+
+The [**DOM Designer**](xref:DOM_Designer) allows you to make quick edits by means of an Excel file. It only supports a limited set of actions:
```

Comment:
Would rephrase this in: The DOM designer is very easy and quick to set up an entire new module with a definition from scratch, to to edits it only supports a limited set of actions. Therefore to do edits to modules & definitions & for more complex modules with multiple definitions reusing sections it is more advised to use the DOM editor.